### DISCUSSION TITLE
Weekly update 6/8/2022

### DISCUSSION THREAD

**Discussion by kirubeltadesse** [Swarmies/sap-core#37] [General]
Please update us using the usual template :)

**Comment by kirubeltadesse** [comment]
# Kirubel

###  How do you feel today?

- Happy and energetic 🔋  

### What did you do since the last meeting? 

- Worked on the docker documentation #27 
- Worked on updating the CICD to use the docker containers #34 
- Merged both #27  and #26 tickets
- Review PRs #32 
- Created more tickets 

### What are you planning to do moving forward? 

- Finishing up the update of the circeCI file.
- Waiting for the team discussion to create a work ticket for each UI, SVC, and DB work
- Planning to start work on #33 ticket once I am done with what I have. 

### Anything hampering your process? 

- Nothing is hampering my progress.

**Comment by dinaolmelak** [comment]
# Dinaol

###  How do you feel today?

- feeling Good. 

### What did you do since the last meeting? 

- Review PRs #32 

### What are you planning to do moving forward? 

- Review more PRs, Test the scaffold and also create a PR.

### Anything hampering your process? 

- Hadn't set time to work on the project on a daily basis. I will allocate time!

**Comment by Getnet12** [comment]
# Getnet

###  How do you feel today?

- Feeling very sad😢😭. I lost one of my friends in a car accident in Dallas, TX. RIP to him. Please drive safe, and may God watch over you guys' journey🙏🏿. 

### What did you do since the last meeting? 

- Revied the PRs #32 

### What are you planning to do moving forward? 

- planning to stay active and look into ticket #35 

### Anything hampering your process? 

- I have been all over the place the last couple of weeks. I'll try to commit enough time.

**Comment by BAbate** [comment]
How do you feel today?

Very thankful 

What did you do since the last meeting?
Submitted  PRs ER modeling  #32

What are you planning to do moving forward?


Correct on feedbacks on ER modeling

Anything hampering your process?
I am working on an internship which is very demanding. I am learning a new frontend framework along with unit testing so I am devoting most of my time to it.
